daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -2205,6 +2215,31 @@ case class DFDLXBitExpr(nameAsParsed: String, fnQName: RefQName,
     res
   }
 }
+
+case class DFDLXBitNot(nameAsParsed: String, fnQName: RefQName,
```

Comment:
I think this should be renamed DFDLXBitNotExpr. DFDLXBitNot should be the name of the runtime class that actually does the bitwise-not.